### PR TITLE
Bugfix for regions validation

### PR DESCRIPTION
**Pull request by bieli**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -174,8 +174,13 @@ func ValidateConf(conf YamlConfig, enableLogicalValidator bool) (bool, map[strin
 	validator.SetValidationFunc("UniqueValuesSlice", UniqueValuesInSliceValidator)
 	valid, validationErrors := validator.Validate(conf)
 	if valid && enableLogicalValidator {
-		conf.RegionsEntryLogicalValidator(&valid, &validationErrors)
-		conf.ListenPortsLogicalValidator(&valid, &validationErrors)
+		var validRegions bool
+		var validListenPorts bool
+		conf.RegionsEntryLogicalValidator(&validRegions, &validationErrors)
+		conf.ListenPortsLogicalValidator(&validListenPorts, &validationErrors)
+		if !validRegions || !validListenPorts {
```
Comment (mjarco) [inline config/config.go]: How about removing this negative if with
```
conf.RegionsEntryLogicalValidator(&valid, &validationErrors)
conf.ListenPortsLogicalValidator(&validListenPorts, &validationErrors)
valid = valid && validListenPorts

```
?

File: log/logger.go [hunk 2]
```diff
@@ -11,10 +11,12 @@ import (
...
 )
 
-const (
-	// ContextreqIDKey is Request Context Value key for debug logging
-	ContextreqIDKey = "reqID"
-)
+// ContextKey for logging context information
+type ContextKey string
+
+func (c ContextKey) String() string {
```
Comment (mjarco) [inline log/logger.go]: What was wrong with old construction. Why you're implementing Stringer for named string type? And then not use this metrod?

Comment (bieli) [inline log/logger.go]: goling + `vet` alerting

File: config/config.go (hunk 1 above)
Comment (bieli) [inline config/config.go]: Ok, we can removing 3 lines from code in your idea

**Pull request merged**